native/internal/math32/sqrt.go

```diff
@@ -0,0 +1,24 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//+build !amd64 noasm
+
+package math32
+
+import (
+	"math"
+)
+
+// Sqrt returns the square root of x.
+//
+// Special cases are:
+//	Sqrt(+Inf) = +Inf
+//	Sqrt(±0) = ±0
+//	Sqrt(x < 0) = NaN
+//	Sqrt(NaN) = NaN
+func Sqrt(x float32) float32 {
+	// FIXME(kortschak): Direct translation of the math
```

Comment:
Note that there is a `Sqrt(float32) float32` in [golang.org/x/mobile/f32](https://godoc.org/golang.org/x/mobile/f32), but this just does what I do here with a TODO. When that happens it will presumably result in the registration of the relevant instructions (or at least an appropriate WORD that we can be confident in).